index.html

```diff
@@ -487,12 +489,7 @@ <h4 data-i18n="index.page.requirements.updates.header">Frequent Updates</h4>
                         url = "https://github.com/adobe/brackets/releases/download/" + tag + "/Brackets." + buildName.split(" ").join(".") + ext;
                     }
 
-                    ///if(OS=="WIN") url = "http://bit.ly/QrVHNc"; // (Temporary) TRACKING WIN
-                    ///else if(OS=="OSX") url = "http://bit.ly/1hQGEsh"; // (Temporary) TRACKING OSX
-
-                    var version_label = buildName.match(/(\d+.*$)/)[1];
-                    //if (OS != "UNKNOWN") version_label += " (" + OS + ")";
-                    $("#download-brackets-version").text(version_label);
+                    $("#download-brackets-version").text(buildNum);
```

Comment:
Is this change intended? The functionality was changed from `version_label` being a match of `\d+.*$` to `buildNum`'s `[\d.]+` match, which are different things.